addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -351,13 +354,36 @@
                 </div>
             </div>
             <div role="tabpanel" t-att-class="comments and 'tab-pane fade in show active' or 'tab-pane fade'" id="discuss">
-                <t t-call="portal.message_thread">
+                <t t-set="enable_add_slide_comment" t-value="0"/>
+                <p t-if="not (slide.channel_id.allow_comment and slide.channel_id.channel_type == 'training')">
+                    Commenting is not enabled on this course.
+                </p>
+                <t t-elif="not slide.comments_count">
+                    <p t-if="not can_access_channel">
+                        There are no comments for now.
+                        <t t-if="slide.channel_id.enroll != 'invite'">
+                            <div class="o_wslides_js_course_join o_wslides_no_access_comments d-inline">
+                                <t t-if="slide.channel_id.enroll == 'public'" t-call="website_slides.join_course_link"/>
+                            </div>
+                            to be the first to leave a comment.
+                        </t>
+                    </p>
+                    <p t-elif="not slide.channel_id.can_comment">
+                        There are no comments for now. Earn more Karma to be the first to leave a comment.
+                    </p>
+                    <t t-else="" t-set="enable_add_slide_comment" t-value="1"/>
+                </t>
+                <t t-else="">
+                    <t t-set="enable_add_slide_comment" t-value="1"/>
```

Comment:
Does that mean we enable the comment section even if the user may not have enough karma?